cornice_swagger/swagger.py

```diff
@@ -145,8 +144,10 @@ def from_path(self, path):
         :type path: string
         :rtype: list
         """
+        path_components = path.split('/')
+        param_names = [comp[1:-1] for comp in path_components
+                       if comp.startswith('{') and comp.endswith('}')]
```

Comment:
The documentation at http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/hybrid.html makes it seem like `*traverse` and `*subpath` don't occur in `{`/`}` but just show up at the end of the path.